### DISCUSSION TITLE
Ajax and Symfony

### DISCUSSION THREAD

**Discussion by Elmatbaisaad** [symfony/symfony#46389] [Q&A]
**<h3>Hello everyone
I wonder if i can use mysql query in ajax in twig ?
thank you</h3>**

**Comment by Nek-** [comment]
Hello, short answer is yes of course.

The complete answer is that you cannot call MySQL with ajax (javascript), but you can call a controller that calls MySQL. And of course you can render your JS with twig, but the recommended way is to use webpack encore.

Learn more here https://symfony.com/doc/current/frontend.html
